distribution/xfer/download.go

```diff
@@ -18,6 +18,8 @@ import (
 
 const maxDownloadAttempts = 5
 
+var waitDuration = time.Second
```

Comment:
I prefer to make this a parameter to `NewLayerDownloadManager` (or a functional argument, if we want to get fancy). Having unit tests change global variables like this can be a problem if we ever want to run tests in parallel.